templates/why-juju.html

```diff
@@ -42,7 +42,7 @@ <h1 class="p-heading--2">It’s time to open source the operations code</h1>
       <p class="p-heading--5">To unlock the true potential of open source, we must normalise open sourcing not just the application code, but also the operational frameworks required to take the most out of it</p>
```

Comment:
Also in the doc = "Juju compared to similar tools"
And in the demo = "How does Juju compare to similar tools"